### DISCUSSION TITLE
How to re-enable E325 (swap file attention prompt)?

### DISCUSSION THREAD

**Discussion by Jorenar** [neovim/neovim#29675] [Q&A] [answered]
In `:h W325` we read:
>The default [SwapExists](https://neovim.io/doc/user/autocmd.html#SwapExists) handler ([default-autocmds](https://neovim.io/doc/user/vim_diff.html#default-autocmds)) skips the [E325](https://neovim.io/doc/user/usr_11.html#E325) prompt
(and automatically chooses "(E)dit") if the swapfile owner process is still
running and owned by the current user.  This presumes that you normally don't
want to be bothered with the [ATTENTION](https://neovim.io/doc/user/usr_11.html#ATTENTION) message just because you happen to
edit the same file from multiple Nvim instances.

Well, that's a wrong presumption in my case. How can I re-enable the prompt asking me what to do when opening a file in multiple Nvim instances?

**Comment by zeertzjq** [comment, marked as answer]
Read the first sentence at `:h default-autocmds`

**Reply by Jorenar** [reply]
OK, brain fart.
Was doing `autocmd! SwapExists` instead of `autocmd! nvim_swapfile`

Thanks!
